js/tests/builders.spec.ts

```diff
@@ -32,7 +33,7 @@ describe('messages builders', () => {
   })
```

Comment:
i think line above this one should now be
```    expect(actual.parentId).to.be.undefined
```

Reply:
thank yewww!